### PR TITLE
Add r-sparkr (SparkR) recipe

### PR DESCRIPTION
**Pull request by zero323**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by zero323** [comment]
I am willing to be listed as a maintainer.

**Comment by dbast** [comment]
hm... would be nice to package it from CRAN as they do some good QA for R packages... but seems like if was removed from there https://cran.r-project.org/web/packages/SparkR/index.html

File: recipes/r-sparkr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+about:
+  home: http://spark.apache.org/
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Provides an R Front end for 'Apache Spark' <https://spark.apache.org>.
+
+extra:
+  recipe-maintainers:
+    - zero323
```
Comment (dbast) [inline recipes/r-sparkr/meta.yaml:43]: Can you please also add the conda-forge/r maintainer group. They generally help with updates and migration of R packages (mostly from CRAN). thx.
```suggestion
    - zero323
    - conda-forge/r
```

**Comment by zero323** [comment]
> hm... would be nice to package it from CRAN as they do some good QA for R packages... but seems like if was removed from there https://cran.r-project.org/web/packages/SparkR/index.html

Yeah....  That would be nice, sadly there were a lot of ups downs when it comes to CRAN, so there is really no guarantee that any specific release will be there, even if submissions are to be continued.

In contrast, deploying to the ASF dist repos is part of the release process.

File: recipes/r-sparkr/meta.yaml (hunk 1 above)
Comment (zero323) [reply recipes/r-sparkr/meta.yaml:43]: Doe.

Comment (zero323) [reply recipes/r-sparkr/meta.yaml:43]: Sure, done.

**Comment by dbast** [comment]
thanks!

**Comment by zero323** [comment]
Thank you @dbast!

**Pull request merged**